google/oauth2/reauth.py

```diff
@@ -324,6 +324,10 @@ def refresh_grant(
     response_status_ok, response_data, retryable_error = _client._token_endpoint_request_no_throw(
         request, token_uri, body
     )
+   
+    if type(response_data) != dict:
+        response_data = {"data": response_data}
+   
```

Comment:
Thanks for fixing this. This was fixed in the oauth2 client code like [so](https://github.com/googleapis/google-auth-library-python/blob/main/google/oauth2/_client.py#L63-L64).

```suggestion
    if isinstance(response_data, six.string_types):
        raise exceptions.RefreshError(response_data, retryable=False)
```

Please make sure to add a unit test for this!

Reply:
Hey @clundin25, I'm not really interested in running with this one. Mostly just wanted to point out the bug.

If the maintainers want to take over from here, that would be great 😊